src/pages/SectionRegistration.tsx

```diff
@@ -83,7 +83,7 @@ function buildSectionRegisterRequest(values: SectionRegistrationFormValues): Sec
             externalIdentifier: sample.externalIdentifier.trim(),
             fixative: labware.fixative.trim(),
             hmdmc: sample.hmdmc.trim(),
-            lifeStage: sample.lifeStage in LifeStage ? sample.lifeStage : undefined,
+            lifeStage: Object.values(LifeStage).includes(sample.lifeStage) ? sample.lifeStage : undefined,
```

Comment:
Does OriginalSampleRegistration have the same problem?

Reply:
I will double check now

Reply:
Same issue occurring on OriginalSampleRegistration and block registration forms, fixed in both